src/util.h

```diff
@@ -31,6 +31,7 @@
 static const bool DEFAULT_LOGTIMEMICROS = false;
 static const bool DEFAULT_LOGIPS        = false;
 static const bool DEFAULT_LOGTIMESTAMPS = true;
+static const int64_t DEFAULT_MAX_TIME_ADJUSTMENT = 70 * 60;
```

Comment:
​Also, the github pull subject line needs update. (It's basically a
different pull)